docker/images/docker-spark/master/startup.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+set -exo pipefail
```

Comment:
won't setting -x show the password in the logs?

Reply:
Yeah I use it just for debugging. I forgot to get rid of it before pushing.